### PR TITLE
[Plugin 1522] Enabling support for the precision less numbers in PostgreSQL plugin.

### PR DESCRIPTION
**Pull request by MrRahulSharma**
Converting the precision less numbers into String type and ensuring that they are compatible with the Postgres Sink as well.
Related JIRA :
https://cdap.atlassian.net/browse/PLUGIN-1522

### CODE REVIEW

File: postgresql-plugin/src/main/java/io/cdap/plugin/postgres/PostgresDBRecord.java [hunk 1]
```diff
@@ -51,22 +53,30 @@ public PostgresDBRecord() {
...
                              int columnIndex, int sqlType, int sqlPrecision, int sqlScale) throws SQLException {
-    if (isUseSchema(resultSet.getMetaData(), columnIndex)) {
+    ResultSetMetaData metadata = resultSet.getMetaData();
+    if (isUseSchema(metadata, columnIndex)) {
       setFieldAccordingToSchema(resultSet, recordBuilder, field, columnIndex);
     } else {
+      // For the precision less number use the string type
+      int columnType = metadata.getColumnType(columnIndex);
+      int precision = metadata.getPrecision(columnIndex);
+      if (columnType == Types.NUMERIC && precision == 0) {
```
Comment (itsankit-google) [inline postgresql-plugin/src/main/java/io/cdap/plugin/postgres/PostgresDBRecord.java]: is this backward compatible like we did for oracle plugins, the user should be able to manually modify/cast the output schema to `decimal`?

Comment (MrRahulSharma) [reply postgresql-plugin/src/main/java/io/cdap/plugin/postgres/PostgresDBRecord.java]: Added the code changes for backward compatibility.

File: postgresql-plugin/pom.xml [hunk 2]
```diff
@@ -87,6 +79,16 @@
       <version>RELEASE</version>
       <scope>compile</scope>
     </dependency>
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-core</artifactId>
+      <scope>compile</scope>
```
Comment (itsankit-google) [inline postgresql-plugin/pom.xml]: `scope` should be `test` for dependencies used only in unit tests.

File: postgresql-plugin/pom.xml [hunk 3]
```diff
@@ -87,6 +79,16 @@
...
+    </dependency>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <scope>compile</scope>
```
Comment (itsankit-google) [inline postgresql-plugin/pom.xml]: similarly here

**Review by itsankit-google — APPROVED** [review]
Minor comments, otherwise LGTM.

**Pull request merged**